actionpack/test/controller/support/custom_assertions.rb

```diff
@@ -0,0 +1,3 @@
+def assert_filtered_out(params, key)
+  assert !params.has_key?(key), "key #{key.inspect} has not been filtered out"
+end
```

Comment:
I'm not sure we need that level of nesting, maybe we could just have something like `ParametersHelper` or something like that?

@senny any thoughts?